workers/api-server/src/http/wallet.rs

```diff
@@ -171,6 +174,16 @@ impl TypedHandler for CreateWalletHandler {
         // and create proofs of validity
         let wallet_id = req.wallet.id;
         let mut wallet: Wallet = req.wallet.try_into().map_err(|e: String| bad_request(e))?;
+
+        // Modify the public of the new wallet to reflect the overwritten fields
```

Comment:
Should be: `// Modify the public shares of the new wallet to reflect the overwritten fields`

Reply:
yes, fixed